keel-sql/src/main/resources/db/changelog/20191218-add-status-to-environment-artifact-versions.yml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-status-to-environment-artifact-versions
+      author: fletch
+      changes:
+        - addColumn:
+            tableName: environment_artifact_versions
+            columns:
+              - column:
+                  name: status
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+      rollback:
+        - dropColumn:
+            tableName: environment_artifact_versions
+            columnName: status
```

Comment:
That's what this is

Reply:
Is this "promotion" status or "artifact" status?